metagpt/provider/bedrock/base_provider.py

```diff
@@ -0,0 +1,29 @@
+import json
+from abc import ABC, abstractmethod
+
+
+class BaseBedrockProvider(ABC):
```

Comment:
any preferable name of `Provider` here to distinguish with folder name provider?

Reply:
I currently don't have any ideas. Bedrock seems more like a transit station for other providers like claude or llama3.